File: app/src/main/java/com/ninhttd/devtest/presentation/product/model/Price.java

package com.ninhttd.devtest.presentation.product.model;

import com.google.gson.annotations.SerializedName;


@SuppressWarnings("unused")
public class Price {

    @SerializedName("sellPrice")
    private Object mSellPrice;
    @SerializedName("supplierSalePrice")
    private Object mSupplierSalePrice;

    public Object getSellPrice() {
        return mSellPrice;
    }

    public void setSellPrice(Object sellPrice) {
        mSellPrice = sellPrice;
    }

    public Object getSupplierSalePrice() {
        return mSupplierSalePrice;
    }

    public void setSupplierSalePrice(Object supplierSalePrice) {
        mSupplierSalePrice = supplierSalePrice;
    }

}

File: app/src/main/java/com/ninhttd/devtest/data/entity/Price.kt
package com.ninhttd.devtest.data.entity

import com.google.gson.annotations.SerializedName


class Price {

    @SerializedName("sellPrice")
    var sellPrice: Any? = null
    @SerializedName("supplierSalePrice")
    var supplierSalePrice: Any? = null

}
